easybuild/easyconfigs/g/GTDB-Tk/GTDB-Tk-2.0.0-intel-2021b.eb

```diff
@@ -0,0 +1,43 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'GTDB-Tk'
+version = '2.0.0'
+
+homepage = 'https://github.com/Ecogenomics/GTDBTk'
+description = "A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes."
+
+toolchain = {'name': 'intel', 'version': '2021b'}
+
+source_urls = ['https://pypi.io/packages/source/g/gtdbtk']
+sources = ['gtdbtk-%(version)s.tar.gz']
+checksums = ['53496443d6fcca675c3f3078d76cef25c2040e894fd403b74397980578ddb815']
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('DendroPy', '4.5.2'),
+    ('matplotlib', '3.4.3'),
+    ('prodigal', '2.6.3'),
+    ('HMMER', '3.3.2'),
+    ('pplacer', '1.1.alpha19', '', True),
+    ('FastANI', '1.33'),
+    ('FastTree', '2.1.11'),
+    ('tqdm', '4.62.3'),
+]
```

Comment:
Based on https://ecogenomics.github.io/GTDBTk/installing/index.html#third-party-software - add [Mash](https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/m/Mash) as a dependency.

Reply:
Oh, nice catch...
Maybe we should also run `gtdbtk check_install` as a sanity check command to catch this, but it currently fails with:

```
The 'GTDBTK_DATA_PATH' environment variable is not defined.
```

Reply:
I spotted while doing  #15315 earlier today.

I tried a few options for a sanity check command, but all of them failed out - either on that env variable not being set or on the data not being as expected if I pointed it at a directory.

Reply:
fixed in 997f868